projects/common/recaptcha/src/recaptcha-data.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpResponse } from '@angular/common/http';
+// import { Http, Response } from '@angular/http';
```

Comment:
Can you remove unused imports please.

Reply:
I only see the one commented import that is unused.
I have removed it.

Thanks,
Ian